base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
 
 ## Prerequisites
 
-To make the best use of this base template, you need a working knowledge of:
+To make the best use of this guide, you should have a working knowledge of:
+
+* [Git](https://git-scm.com/): The version control system used.
+* [GitHub](https://github.com/thegooddocsproject): The project hosting platform used, along with associated project management tools.
+* [GitHub Flavored Markdown](https://github.github.com/gfm/): The wiki format used to store template documentation.
+* [Currently implemented doctypes templates](https://github.com/thegooddocsproject/templates): To help identify if you need to create a new template or edit an existing one.
+* Technical writing: To produce a well-written document that includes the relevant references.
+
+## Target audience for templates
```

Comment:
On Tue, Jan 5, 2021 at 8:46 AM S... wrote:
Hallo Cameron

Thanks - I had a good read. That's coming along very nicely! My one suggestion would be to reduce the number of audience types, if possible. Simpler is better, even if it means sacrificing some nuance. For example, maybe you don't need the two "reviewer" roles and the "doc reader" role. They're important roles, but it's not clear to me how a template would include tips for those roles.

Cheers

S

Reply:
I agree with S in that we can reduce the number of audience types and thereby simply the base template. 
Here’s what I suggest:

**Template author** -  merging the roles of template author, reviewer and customizer into one, because essentially, all these roles will access the Base Template with the requirement of editing/authoring a new template. 


**Document Reader** - from the point of view of anyone accessing the Base template, we can simplify the tips just for a reader and an author (above).

Reply:
+1 to that. If you're looking for another way to collapse roles, you could have "Template creator" to emcompass the three subroles listed here, "Docset owner" (if it needs to be separately called out), and "Document creator." The TA for the template isn't the reader, but what's produced is for the reader. Since you have a user-facing TA section later, having that called out here might conflate this for some readers?

Reply:
Thanks for the suggestions, I agree it improves things. I've trimmed the roles back.